test/twemoji.js

```diff
@@ -369,7 +369,7 @@ const twemoji = new Set([
   "1f333",
   "1f334",
   "1f335",
-  "1f336-fe0f",
```

Comment:
I guess this could have potential presentation differences without `-fe0f` but probably not a big deal

https://stackoverflow.com/questions/38100329/what-does-u-ufe0f-in-an-emoji-mean-is-it-the-same-if-i-delete-it

Reply:
I just needed a new way to generate the test data, since the old preview.html link was broken.
I think it's fine. It caught things like v15 emojis not being supported.